tests/pytorch/test_parallel_cross_entropy.py

```diff
@@ -89,7 +89,7 @@ def one_iteration_test(
         # Check that loss and grad input match
         tols = dtype_tols(dtype)
         test_loss = test_loss.to(dtype=torch.float64, device="cpu")
-        ref_loss = test_loss.to(dtype=torch.float64, device="cpu")
+        ref_loss = ref_loss.to(dtype=torch.float64, device="cpu")  # FIX: was test_loss
```

Comment:
These kinds of unnecessary comments have negative value once the code is merged in. The right place to explain line-by-line changes is in the PR description, not in the code itself.
```suggestion
        ref_loss = ref_loss.to(dtype=torch.float64, device="cpu")
```